account_budget_oca/models/account_budget.py

```diff
@@ -148,8 +148,8 @@ def _compute_practical_amount(self):
                     SELECT SUM(amount)
                     FROM account_analytic_line
                     WHERE account_id=%s
-                        AND (date between to_date(%s,'yyyy-mm-dd')
-                        AND to_date(%s,'yyyy-mm-dd'))
+                        AND (date between %s
+                        AND %s)
```

Comment:
Looking at [these lines](https://github.com/OCA/account-budgeting/blob/12.0/account_budget_oca/models/account_budget.py#L167-L170) it seems that `wizard_date_from` and `wizard_date_to` are expected to be strings. In this case the `to_date()` is still necessary.
I think you should keep the `to_date()` and convert the `line.date_from` and `line.date_to` to string.

Reply:
Another solution would be to remove all the occurrences of `wizard_date_from` and `wizard_date_to`. It seems they are not used, probably they are a leftover from old code. This way you could keep your solution that removes the `to_date()`.

Reply:
OK then! Let's go with the fix you proposed. But please remove all the occurrences of `wizard_date_from` and `wizard_date_to` because they are not used in the code and in any case they would not work.

Reply:
@cvinh @astirpe hello, I was looking at this module and received the error this issue was created for, is this issue stale or is it still being worked?